tests/test_example_screencast.py

```diff
@@ -0,0 +1,60 @@
+import os
+import filecmp
+import autoscrub
+from pathlib import Path
+
+TEST_VIDEO_INPUT = "example_screencast.mp4"
+TEST_VIDEO_OUTPUT = "example_screencast_scrub.mp4"
+VALID_FILTERGRAPH = "example_screencast.txt"
+VALID_DURATION = 48.57
+VALID_LOUDNESS = {
+    "I": -18.1,
+    "Threshold": -39.2,
+    "LRA": 7.3,
+    "LRA low": -23.5,
+    "LRA high": -16.2,
+}
+
+os.chdir(Path(__file__).parent.absolute())
+
+
+def test_autoprocess(input=TEST_VIDEO_INPUT, filtergraph=VALID_FILTERGRAPH):
+    """Runs autoscrub autoprocess --debug on TEST_VIDEO_INPUT
+
+    Checks stdout and compares resulting filtergraph to known valid version
+    """
+    # Create output path
+    suffix = Path(input).suffix
+    output = input.replace(suffix, "_scrub" + suffix)
+
+    # Run autoscrub autoprocess --debug input output
+    p = autoscrub._agnostic_Popen(
+        ["autoscrub", "autoprocess", "--debug", input, output]
+    )
+    _, stderr = autoscrub._agnostic_communicate(p)
```

Comment:
Per #2 in which processing continued indefinitely, a timeout is essential here, akin to [`subprocess.Popen.communicate`](https://docs.python.org/3/library/subprocess.html#subprocess.Popen.communicate). @philipstarkey, is this something that could/should be added to `autoscrub._agnostic_communicate` or should I be using something else. here?

Reply:
For now I've set a timeout on the test runner, using [`pytest-timeout`](https://github.com/pytest-dev/pytest-timeout/).

Reply:
Not sure why, but the test is stalling on Windows at [this point](https://github.com/rpanderson/autoscrub/runs/660407925?check_suite_focus=true#step:8:133) (both locally and on Python 2.7 and 3.8).